retrofit-adapters/rxjava/src/main/java/retrofit2/adapter/rxjava/Transformer.java

```diff
@@ -0,0 +1,15 @@
+package retrofit2.adapter.rxjava;
+
+import rx.Completable;
+import rx.Observable;
+
+/**
+ * A class that allows {@link Observable} and {@link Completable} to be transformed before
+ * returning then from {@link RxJavaCallAdapterFactory}.
+ *
+ * For example, you can use the {@link Transformer} to apply a observeOn or subscribeOn {@link rx.Scheduler}
+ */
+public interface Transformer {
```

Comment:
PS `Single` is supported because `RxJavaCallAdapterFactory` uses the same `CallAdapter` as used by `Observable` and just calls `toSingle` on it